### ISSUE TITLE
Authentication Failed with odin login -c --m3 <https://m3> <file.ionapi>

### ISSUE THREAD

**Issue by ismail-spoon** [infor-cloud/m3-h5-sdk#174]
**Describe the bug**
Authorization app has been created exactly like in the example in my Infor IONAPI environnement. 
After the entering the login in the pop up and approving the application, a an error is shown:

Logon request failed (urn:oasis:names:tc:SAML:2.0:status:Responder - STATUS-MISSING - Unexpected Runtime Authn Adapter Integration Problem.) (error: 612)
![image](https://github.com/infor-cloud/m3-h5-sdk/assets/95227711/ac8c4adb-e8e0-4d29-a10a-4f45bc3e1c07)

Authorization application: 
Port 10000 because 8080 is blocked on my PC.
![image](https://github.com/infor-cloud/m3-h5-sdk/assets/95227711/68828689-bda5-431f-a25d-6eb2d54878d6)

**To Reproduce**
1. Go to 'odin login -c --m3 <https://m3> <file.ionapi>'
2. Enter login informations
3. Click on approve

**Expected behavior**
It should fetch the cookie and save the session.

**Version**
 - `@infor-up/m3-odin`: 6.0.0
 - `@infor-up/m3-odin-angular`: 6.0.0
 - `@infor-up/m3-odin-cli`: none

**Comment by swuendri** [comment]
Hello @ismail-spoon, are you in a multi tenant cloud environment? If yes, please omit the --m3 option.

**Comment by ismail-spoon** [comment]
> Hello @ismail-spoon, are you in a multi tenant cloud environment? If yes, please omit the --m3 option.

Yes I am on a MT environment. I'm trying  to resolve this issue as when I deploy the H5 application on the MT environment the app works, but not on local development. Some APIs are working so I think it is something different. Any ideas?:
![image](https://github.com/infor-cloud/m3-h5-sdk/assets/95227711/8c93b30e-998d-44d8-9b2e-c6c7d436b89c)

**Comment by ismail-spoon** [comment]
Hello, yes I was able to login using these commands in order:
`odin login --update-config ./src/assets/ionapi/DEV.ionapi`
`odin serve --multi-tenant --ion-api --port 9000`

The serving works, but it gives me the above error at some point, is there something in this configuration (odin.json) that needs to be changed? 
![image](https://github.com/infor-cloud/m3-h5-sdk/assets/95227711/8ce2b1db-a929-4920-8e45-55e3130a3d05)

Also I found out that by adding the below extension the error is fixed. It is a workaround atm but what could be the actual solution. 
![image](https://github.com/infor-cloud/m3-h5-sdk/assets/95227711/6381eb30-a7d5-4ec7-b2d1-798230edc16d)

**Comment by swuendri** [comment]
Hello @ismail-spoon, how can I reproduce the CORS error on my site?
